authBackEnd/apps/common/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
There's a bunch of empty files -> would it be an issue if they were deleted?

Reply:
Completed